docs/samples/Microsoft.ML.Samples/Dynamic/ConvertType.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ConvertTypeTransform
+    {
+        private sealed class InputData
+        {
+            public bool Survived;
+        }
+
+        private sealed class TransformedData
+        {
+            public bool Survived { get; set; }
```

Comment:
>{ get; set; } [](start = 33, length = 13)

do those need to be properties?

Reply:
setting it to properties helps me get away with warning about uninitialized field

---
In reply to: [261046835](https://github.com/dotnet/machinelearning/pull/2781#discussion_r261046835) [](ancestors = 261046835)